docs/Tutorial%3A-How-to-Configure-the-SMART-on-FHIR-Sample-Application-using-OSS-FHIR-Server.md

```diff
@@ -0,0 +1,55 @@
+You can follow the steps outlined in the document, "[Tutorial: Azure Active Directory SMART on FHIR proxy](https://docs.microsoft.com/en-us/azure/healthcare-apis/use-smart-on-fhir-proxy)." However, there are a few differences, as documented here.
```

Comment:
I would reword the introduction like this:

To configure the SMART on FHIR sample application using the open-source FHIR Server for Azure you can follow most of the instruction sin the document "Tutorial...". Below we have highlighted where you will need to take alternate steps.